src/bindings/python/src/openvino/__init__.py

```diff
@@ -3,3 +3,8 @@
 # SPDX-License-Identifier: Apache-2.0
 
 __path__ = __import__("pkgutil").extend_path(__path__, __name__)  # type: ignore # mypy issue #1422
+
+try:
+    from openvino.tools.mo.convert import convert
```

Comment:
Is it a final decision to have it in `openvino`?

Reply:
Not sure that it is final decision. @slyalin please take a look. 

I also moved LayoutMap and InputCutInfo to openvino.convert package. I am not sure that it is good to have both openvino.convert function name and openvino.convert package name. In simple test it works, but if we import LayoutMap or InputCutInfo prior to convert method python starts to treat convert as module and it does not work. @slyalin please comment. Maybe we should move LayoutMap,InputCutInfo to tools.mo.convert? In this case there is no problem.

Reply:
@slyalin has confirmed that it is final decision.

Reply:
Not completely. We just want to understand if having it at this level simplifies user code and in the same time doesn't involve any mess in imports. It is a proposed shortcut.

Reply:
It was decided to leave place convert() only in openvino.tools.mo for this PR. There will be discussion on adding alias of convert in more convenient place, it will be implemented in a separate PR.